flake.nix

```diff
@@ -1,77 +1,144 @@
 {
-  description = "Template for Holochain app development";
+  description = "Flake for Holochain app development";
 
   inputs = {
-    nixpkgs.follows = "holochain/nixpkgs";
-    versions.url = "github:holochain/holochain?dir=versions/weekly";
+    holonix.url = "github:holochain/holonix?ref=main";
 
-    holochain = {
-      url = "github:holochain/holochain";
-      inputs.versions.follows = "versions";
+    nixpkgs.follows = "holonix/nixpkgs";
+    flake-parts.follows = "holonix/flake-parts";
+
+    crane = {
+      url = "github:ipetkov/crane";
+      inputs.nixpkgs.follows = "holonix/nixpkgs";
+    };
+
+    rust-overlay = {
```

Comment:
Holonix will also be pulling in the rust-overlay and crane, one should follow the other.

I am currently preferring that dependencies which use the same inputs as you should be made to follow your inputs - see Wind Tunnel or tryorama for the way the follows have been set up there.

This is optional but I think it's cleaner to use one toolset

Reply:
I see, I'll leave it as is for now, will update once I have a better understanding of how the new holonix is set up.

Reply:
Cool, yeah it's nothing urgent. Happy to talk it through on a call because it took me a while to get my head round why I was seeing people do this in public flakes